### PR TITLE
[WIP] Add cluster logging (only for testing for now)

### PR DESCRIPTION
**Pull request by avinal**

### CODE REVIEW

File: operator/gitops/argocd/logging-operator/olo-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-logging
```
Comment (adambkaplan) [inline operator/gitops/argocd/logging-operator/olo-namespace.yaml:5]: Ditto - link to docs. https://docs.openshift.com/container-platform/4.11/logging/cluster-logging-deploying.html#cluster-logging-deploy-cli_cluster-logging-deploying

File: operator/gitops/argocd/logging-operator/olo-og.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
```
Comment (adambkaplan) [inline operator/gitops/argocd/logging-operator/olo-og.yaml:3]: Ditto - link to docs

**Comment by avinal** [comment]
@adambkaplan this PR is a just for test purpose, I am trying to put the pieces together and then I will update it as per our requirements.

**Comment by avinal** [comment]
Obsolote

**Pull request closed without merging**